paperqa/agents/search.py

```diff
@@ -358,23 +356,22 @@ async def process_file(
             if file_name in metadata:
                 doi, title = metadata[file_name].doi, metadata[file_name].title
 
-            # note extras are forbidden in docs
-            chunk_chars, overlap = 3000, 250
-            if "chunk_chars" in docs_kwargs:
-                chunk_chars = int(docs_kwargs.pop("chunk_chars"))
-            if "overlap" in docs_kwargs:
-                overlap = int(docs_kwargs.pop("overlap"))
-
-            tmp_docs = Docs(**docs_kwargs)
+            # We set the LLM/SummaryLLM because
+            # it could be that all metadata fails and it has to peek to first page
+            # TODO: Eventually we will have an LLM object of some kinda passed here
+            # (or coming from settings)
```

Comment:
Is this actually a `TODO`? I don't see anything being "to do" here

Also, typo in `kinda`